src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/depth_to_space/depth_to_space_kernel_ref.cpp

```diff
@@ -40,8 +40,15 @@ CommonDispatchData DepthToSpaceKernelRef::SetDefault(const depth_to_space_params
                          params.outputs[0].Feature().v,
                          params.outputs[0].Z().v * params.outputs[0].Y().v * params.outputs[0].X().v };
 
-    // this kernel only supports bfyx and b_fs_yx_fsv16 layout.
+    //The reason why reverse input/output of GetOptimalLocalWorkGroupSizes():
+    //Large X*Y*Z lws size is better than large batch lws, but current GetOptimalLocalWorkGroupSizes not work like that.
```

Comment:
nit: whitespace between `//` and actual comment.

Reply:
Done.